### DISCUSSION TITLE
pyMOR interface with FEM software

### DISCUSSION THREAD

**Discussion by Andre0809** [pymor/pymor#1433] [Q&A]
For FEM dynamic simulation with Ansys or Abaqus, etc, when the DOF of model increases to million scales, it may  need lots time for simulation. take the 2nd order system: forced vibration system as example. when the excitation force amplitude or frequency is changing, it's expected to get the response. Each simulation would be time consuming. So if we could get the ROM through the algorithm inside pyMOR, then the matrix of mass, stiffness, damping would be reduced from DOF of N to m, while m<<N, during this process we need a interface to read the output files of FEM software to get the full matrix, also define the input and output of system; 2nd step is to run the MOR algorithm to get reduced model or matrix; 3rd step is run the specific input to get the new output.
Currently such kind of tool available for engineering is few, but it's increasing demand as the Digitial Twin and IoT boosting in industry and life.

**Comment by renefritze** [comment]
Are you asking whether there are pyMOR bindings for Ansys/Abaqus?
